app/src/main/java/com/nervousfish/nervousfish/data_objects/Database.java

```diff
@@ -0,0 +1,107 @@
+package com.nervousfish.nervousfish.data_objects;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+
+import java.io.InvalidObjectException;
+import java.io.ObjectInputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A database object to write and encrypt to file
+ */
+public final class Database implements Serializable {
+    private static final long serialVersionUID = -4715364587956219157L;
+    private final List<Contact> contacts;
+    private Profile profile;
```

Comment:
Profile MUST be Serializable, otherwise you can get a very nasty bug causing Database not to be Serializable